manifests/server/config.pp

```diff
@@ -139,11 +139,7 @@
     require => Exec['puppet_server_config-create_ssl_dir'],
   }
 
-  file { "${::puppet::server::ssl_dir}/private_keys/${::puppet::server::certname}.pem":
-    owner => $::puppet::server::user,
-    group => $::puppet::server::group,
-    mode  => '0640',
-  }
+  ensure_resource('file', "${::puppet::server::ssl_dir}/private_keys/${::puppet::server::certname}.pem", {ensure => file})
```

Comment:
Yes, we have a module that handles all our ssl keys and certificates. I will change the pull-request and add a switch to disable the certificate handling, so that the default behavior stays the same and we can toggle it off the let our module do the work.